p2pserver/message/types/block.go

```diff
@@ -48,8 +53,18 @@ func (this *Block) Deserialization(source *common.ZeroCopySource) error {
 	if err != nil {
 		return fmt.Errorf("read Blk error. err:%v", err)
 	}
-
-	eof := false
+	hasEmptyCCM, irr, eof := source.NextBool()
```

Comment:
need compatible with old version node.

Reply:
you can see the Serialization method,  
```sink.WriteBool(this.CCMsg != nil)
	if this.CCMsg != nil {
		this.CCMsg.Serialization(sink)
	}```
if CCMsg is nil, it will write false, when you Deserialization, it will get false.